x-pack/platform/plugins/shared/cases/server/client/attachments/delete.ts

```diff
@@ -163,3 +177,42 @@ const handleAlerts = async ({ alertsService, attachments, caseId }: HandleAlerts
   const alerts = getAlertInfoFromComments(alertAttachments);
   await alertsService.removeCaseIdFromAlerts({ alerts, caseId });
 };
+
+interface UpdateCaseAttachmentStats {
+  caseService: CasesClientArgs['services']['caseService'];
+  attachmentService: CasesClientArgs['services']['attachmentService'];
+  caseId: string;
+  user: CasesClientArgs['user'];
+}
+
+const updateCaseAttachmentStats = async ({
+  caseService,
+  attachmentService,
+  caseId,
+  user,
+}: UpdateCaseAttachmentStats) => {
+  const originalCase = await caseService.getCase({
+    id: caseId,
+  });
+
+  const date = new Date().toISOString();
+
+  const attachmentStats = await attachmentService.getter.getCaseAttatchmentStats({
+    caseIds: [caseId],
+  });
+
+  const totalComments = attachmentStats.get(caseId)?.userComments ?? 0;
+  const totalAlerts = attachmentStats.get(caseId)?.alerts ?? 0;
+
+  await caseService.patchCase({
+    originalCase,
+    caseId,
+    updatedAttributes: {
+      updated_at: date,
+      updated_by: user,
```

Comment:
I noticed we include these two `updated_*` here, and wanted to check if maybe we were doing that twice.

We create the user action, but never change the `updated` fields. Weren't we updating them when deleting attachments?

Reply:
We do not update these fields when creating user actions. Not sure if we should or not. We wanted to avoid version conflicts that will be caused by updating the case SO as much as possible. But now, with the new functionality, it is inevitable. We need to update the stats. Thought, that makes me think, deleting or creating a comment, is it considered an update to the case? Alternatively, we could only update the stats, but not the `updated_*` fields, as updating them may lead to wrong assumptions about who updated the case last and when.  Wdyt?